tests/CommonTest.php

```diff
@@ -12,8 +12,6 @@ final class CommonTest extends CIUnitTestCase
     protected function setUp(): void
     {
         parent::setUp();
-
-        config('Paths')->viewDirectory = SUPPORTPATH . 'Views';
```

Comment:
Please restore these lines. These tests are run only for this package development, so enabled caching will not affect it.